### DISCUSSION TITLE
Inconsistent mod overlay tooltips & some suggestions

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#19634] [Ideas]
normal mod tooltip:
![osu!_kRoVfo0kPy](https://user-images.githubusercontent.com/43358824/183311429-97795500-aa4a-4eb4-8adf-139c35f3b4ac.png)
personal preset tooltip:
![osu!_VbVqChN8yY](https://user-images.githubusercontent.com/43358824/183311432-88b4daeb-7269-4e34-9368-95092c752c29.png)

currently the normal mod tooltip shows the description of the mod + the incompatible mods using the mod icon version rather than the acronym and it uses a different tooltip bg colour. i think the personal preset tooltip could be improved by:

- changing the background colour to the same as the normal mod tooltip
- using the mod icons rather than the acronyms 
- adding the personal preset description to the tooltip (which could also be helpful if the description is too long to fit)

**Comment by bdach** [comment]
- Aware of the colour mismatch. Was going to address it at some point. The preset tooltip's background is more correct as it follows the theme.
- I like the acronyms better than the icons if I'm honest. If anything I'd be replacing the incompatibility panel's way of doing things in favour of using the acronyms, personally. It really irks me that you have to unmap the incompatibility tooltip icons to mods rather than, you know, just read the names.

  Also remember that the mod preset panel will show mod settings, which the incompatibility tooltip never has to do. And I utterly loath how the score tooltip does this, which is by just putting a naked icon on one line and then several lines of string setting text underneath.
- Unsure what to do with the descriptions. The only time they're useful is when the text is truncated. Other than that they're just repeating information. And detecting when the descriptions truncate isn't easy.
